configure.ac

```diff
@@ -177,8 +177,8 @@ AX_OPTION_GATTRIB
 AX_DATA_DIRS
 # Where should PCB footprints be searched for?
 AX_PCB_DIRS
-# Set up libgeda with the correct ld version number
-AX_LIBGEDA([45:0:0])
+# Set up liblepton with the correct ld version number
+AX_LIBLEPTON([45:0:0])
```

Comment:
@peter-b Yes, though I thought it would be another commit to underline this change. Do you think I should merge this changes in the same commit?